actionview/test/template/form_helper_test.rb

```diff
@@ -2058,6 +2058,13 @@ def test_form_for_with_format
   end
 
   def test_new_form_for_without_format
+    # if the path is to be overridden in an application, make sure
+    # {format:nil} will not be passed in which will break the app
+    def post_path(id, car = nil)
```

Comment:
This method end up leaking to all the test. It is the same as defining outside of the test. I think we need to find another way to test this.